### PR TITLE
[ur] Extend CTS coverage for usm/queue/memory

### PR DESCRIPTION
**Pull request by veselypeta**

### CODE REVIEW

**Review by kbenzie — APPROVED** [review]
LGTM

File: source/loader/ur_ldrddi.cpp [hunk 1]
```diff
@@ -5295,5 +5296,5 @@ urGetDeviceProcAddrTable(
 }
 
 #if defined(__cplusplus)
-};
+}
```
Comment (omarahmed1111) [inline source/loader/ur_ldrddi.cpp:5299]: I think this was covered in this [PR](https://github.com/oneapi-src/unified-runtime/pull/278/files)

Comment (veselypeta) [reply source/loader/ur_ldrddi.cpp]: I've rebased now :)

**Review by martygrant — APPROVED** [review]
Looks good.

**Review by omarahmed1111 — APPROVED** [review]
LGTM

**Pull request merged**